### PR TITLE
DMN plugins support both cloud and platform

### LINKED ISSUE #2854: DMN plugins support (both Cloud and Platform)
* [ ] I can write a plugin for both Cloud and Platform DMN.
* [ ] I can write a plugin specifically for Cloud DMN.
* [ ] I can write a plugin specifically for Platform DMN.
* [ ] I can write a properties panel extension for DMN (cloud/platform/both) 
* [ ] I can check in menu plugin whether the tab is for Cloud or Platform

### PR DESCRIPTION
**Pull request by barmac**
This will be followed up with a PR in the plugin helpers repo.

Closes #2854

### CODE REVIEW

**Comment by barmac** [comment]
Follow-up PR: https://github.com/camunda/camunda-modeler-plugin-helpers/pull/24

File: resources/plugins/test-editor-events/client/TestEditorEvents.js [hunk 1]
```diff
@@ -111,3 +124,23 @@ function addModule(extensionModule) {
     };
   };
 }
+
+function addDmnModule(extensionModule) {
```
Comment (Skaiir) [inline resources/plugins/test-editor-events/client/TestEditorEvents.js:128]: Not a comment with regards to this PR, but yet another example that supports making this function part of the plugin helpers (https://github.com/camunda/camunda-modeler-plugin-helpers/issues/22).

File: client/src/app/tabs/dmn/util/configure.js [hunk 2]
```diff
@@ -9,7 +9,7 @@
  */
 
 export default function configureModeler(
```
Comment (Skaiir) [inline client/src/app/tabs/dmn/util/configure.js:11]: Should we consider moving this file to a dmn-shared folder, like we've done for bpmn to eventually down the line refactor dmn and bpmn editors to minimize code duplication, or would you say cloud and platform dmn are too similar to warrant such a change?

Comment (barmac) [reply client/src/app/tabs/dmn/util/configure.js:11]: At the moment, we import directly from the `dmn` folder, so regarding the configure function, this is not really code duplication. I think we can consider having a shared folder in the future, but it's not absolutely necessary at the moment.

Comment (Skaiir) [reply client/src/app/tabs/dmn/util/configure.js:11]: Alright. with code duplication I was more referring to the DmnEditor classes, like BpmnEditor classes as well. But okay that's not a problem, reducing code duplication in these classes is something I want to have a go at in the future anyways. Maybe a slack time project but you're right it's not absolutely necessary.

**Review by Skaiir — CHANGES_REQUESTED** [review]
@barmac This looks good to me overall, can't see any real problems. Only need to squash those commits and I think this can go in.

**Pull request merged**